src/modules/emote_menu/stores/emote-menu-store.js

```diff
@@ -33,13 +34,38 @@ function computeScore({totalUses, recentUses}) {
   return Math.floor((totalUses * frecency) / recentUses.length);
 }
 
+const TWITCH_EMOTE_USAGE_HISTORY_KEY = 'twilight.emote_picker_history';
+
+function loadDefaultFrequentEmotes() {
+  if (getPlatform() !== PlatformTypes.TWITCH || !storage._localStorageSupport) {
```

Comment:
keys prefixed with underscore are generally considered private and should not be accessed out of the class.